huff_cli/src/huffc.rs

```diff
@@ -204,6 +208,60 @@ fn main() {
         file_provider: Arc::new(FileSystemFileProvider {}),
     };
 
+    if cli.label_indices {
+        match compiler.grab_contracts() {
+            Ok(contracts) => {
+                if contracts.len() > 1 {
+                    eprintln!(
+                        "{}",
+                        Paint::red("Multiple contracts found. Please specify a single contract and try again.")
+                    );
+                    std::process::exit(1);
+                }
+
+                if let Some(contract) = contracts.first() {
+                    let m_macro = contract.find_macro_by_name("MAIN").unwrap();
```

Comment:
There is an alternative macro cli flag you can use here to support any macro

Reply:
added support for alt-main